### ISSUE TITLE
NVDA does not read the user control window

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#6236]
Good Morning.
Today I made the migration from Windows 10 64-bit upgrade 14393.
Also upgraded NVDA to master 13481.
Ever happens that the warning window uac user control requesting permission to run a program is not being read by NVDA.
I read the file what NVDA again there is an error related to not reading this window was fixed, and as on my machine did not read I decided to send this error.
The user control is set to 34%.

**Comment by josephsl** [comment]
Hi, this is a known problem. You should set UAC level to level 2 (67 percent). Thanks.

**Comment by fernando-jose-silva** [comment]
thank you so much

 

De: Joseph Lee [mailto:notifications@github.com] 
Enviada em: quinta-feira, 4 de agosto de 2016 10:39
Para: nvaccess/nvda <nvda@noreply.github.com>
Cc: fernando-jose-silva <fernandoj.silva21@gmail.com>; Author <author@noreply.github.com>
Assunto: Re: [nvaccess/nvda] NVDA does not read the user control window (#6236)

 

Hi, this is a known problem. You should set UAC level to level 2 (67 percent). Thanks.

**Comment by ehollig** [comment]
Is this still an issue with the latest Windows 10? CC @fernando-jose-silva @josephsl

**Comment by fernando-jose-silva** [comment]
Yes build 15063.540.
Thank you

Enviado do meu iPhone

**Comment by bdorer** [comment]
This is still the case with NVDA next 14456,d8cc508f and Windows 
10.0.15063 workstation.



Am 12.09.2017 um 18:15 schrieb Ethan Holliger:

**Comment by Adriani90** [comment]
@fernando-jose-silva, @bdorer is this still an issue in NVDA 2019.1 Beta 1?

**Comment by fernando-jose-silva** [comment]
The problem still occurs, but only with the uac user access control at 34%.
Updated updates, nvda alfa 16885 and windows 10 17763.348

**Comment by bdorer** [comment]
This problem exists with UAC set to 34%. If you set UAC level higher all 
works. I tested with latest next version and Windows 10.0.17763.

Am 08.03.2019 um 23:27 schrieb Adriani90:

**Comment by RichCaloggero** [comment]
I'm running win10 enterprise and I'm seeing odd behavior even with uacct settings higher than 33%.

- window title is announced
- tabbing sometimes works, sometimes not
- sometimes alt+u brings me to username field,, but then tab will not move me to password; need to type alt+u or alt+p to get there
- sometimes after correct credentials entered and alt+y is pressed, system becomes unresponsive for up to a minute
   + during this time I sometimes hear something like "$$$", followed by stuff I can't remember now

**Comment by Adriani90** [comment]
@fernando-jose-silva, @bdorer, @RichCaloggero are you also having this issue with NVDA 2025.1.2 RC and Windows 11 24 H2?

**Comment by bdorer** [comment]
Sorry, I don't use Windows 11.
